theme-settings.php

```diff
@@ -35,4 +40,18 @@ function material_admin_form_system_theme_settings_alter(&$form, \Drupal\Core\Fo
     '#description' => t('Status messages use google material toast notifcation system, but this limits the length shown in the status message. Note: the full message will always appear in the bottom drawer'),
     '#default_value' => theme_get_setting('material_admin_message_length'),
   );
+
+    $form['theme_ui_options']['material_admin_jqueryui_dialog_background'] = array(
```

Comment:
The only reason I have it configurable is that sometimes contrib modules have weird admin UI pages that potentially could conflict. 

I don't want my theme to ever force someone to switch themes to use a module, so providing an option to disable as a short-term solution until the module developer can fix the issue.

It will be on by default (reminds me, I need to add that to the settings.yml)